README.md

```diff
@@ -7,7 +7,8 @@ among the community.
 
 Translations of the guide are available in the following languages:
 
-* [Chinese Simplified](https://github.com/aseaday/git-style-guide)
+* [Simplified Chinese](https://github.com/aseaday/git-style-guide)
```

Comment:
I would like to have the translations sorted alphabetically by the corresponding language. That said, I see that it looks strange to have them like they are right now. We could have them like so:

```
* Chinese (Simplified)
* Chinese (Traditional)
```

P.S. Great to see yet another Chinese translation!